rest/routes/recruitment.py

```diff
@@ -56,5 +58,33 @@ def get_all_recruitment_data():
         return response
 
     data = dict()
-    data['data'] = Recruitment.to_json()
+    data['data'] = [Recruitment.to_json(rec) for rec in Recruitment.query.all()]
+    return create_response(data, 200, '*')
+
+
+@recruitment_url.route('/<recruitment_id>', methods=['GET', 'OPTIONS'])
+def get_recruitment_with_candidates(recruitment_id):
+    """
+    Get recruitment with given id and all candidates
+    :param id: id of recruitment
+    :return: flask Response containing json with recruitment data
+    """
+    role, response = handle_request_token(request)
+
+    if role is None:
+        return response
+
+    try:
+        recruitment = Recruitment.query.filter_by(id=recruitment_id).first()
+        if not recruitment:
+            return create_response({"error": "Nie znaleziono podanej rekrutacji"}, 404, '*')
+        data = dict()
+        data = Recruitment.to_json(recruitment)
+        data['candidates'] = [CandidateRecruitment.to_json(rec) for rec in
+                              recruitment.candidate_recruitments]
+    # pylint: disable=broad-except
+    except Exception as exception:
+        print(exception, file=sys.stderr)
+        return create_response({"error": "Błąd podczas pobierania kandydatów."}, 400, '*')
```

Comment:
https://github.com/cometio-solutions/recio-backend/pull/23#discussion_r613505672
do this here as well